tests/integration/kubernetes/runtimeclass_workloads/pod-file-volume.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       type: File
   containers:
   - name: busybox-file-volume-container
-    image: busybox
+    image: quay.io/prometheus/busybox:latest
```

Comment:
Is this change (and the one below) directly related to this PR?

Reply:
It was. I've added a fix and is no longer needed to pass full image reference, I've reverted this change